client/quantum_serverless/core/provider.py

```diff
@@ -273,6 +276,7 @@ def __init__(
         name: str,
         host: Optional[str] = None,
         namespace: Optional[str] = "default",
+        img: Optional[str] = "qiskit/quantum-serverless-ray-node:latest-py39",
```

Comment:
My only worry about this is that it's a place that we will need to remember to change when it involves a change in the image. How do you see move it to the [constants file](https://github.com/Qiskit-Extensions/quantum-serverless/blob/main/client/quantum_serverless/core/constrants.py) by now, @psschwei , @IceKhan13 ?

Probably not the best solution either but I couldn't think in a better approach by now (cost-effective I mean).

Reply:
I think I'm fine with that... any opinions on whether the constant should be the tag only or the whole thing? (I lean slightly towards the former but don't have a strong preference)

Reply:
Yeah, I don't have a strong opinion either. For me it sounds better the first option that you are mentioning too 👍

Reply:
@psschwei I don't know if you want to address this to use the tag instead of the whole string but from my part is approved. Feel free to merge it 🙏

Reply:
of course, I did the opposite in my patch... do what I do, not what I say huh :laughing: 

> Feel free to merge it

I think we need @IceKhan13 to approve (since he originally requested changes)

Reply:
oh, yes, we can wait then!